code/modules/antagonists/heretic/magic/mind_gate.dm

```diff
@@ -0,0 +1,42 @@
+/datum/action/cooldown/spell/pointed/mind_gate
+	name = "Mind Gate"
+	desc = "Deals you 20 brain damage and the target suffers a hallucination, \
+			is left confused for 10 seconds, and suffers oxygen loss and brain damage."
+	background_icon_state = "bg_heretic"
+	overlay_icon_state = "bg_heretic_border"
+	button_icon = 'icons/mob/actions/actions_ecult.dmi'
+	button_icon_state = "mind_gate"
+
+	sound = 'sound/magic/curse.ogg'
+	school = SCHOOL_FORBIDDEN
+	cooldown_time = 20 SECONDS
+
+	invocation = "Op' 'oY 'Mi'd"
+	invocation_type = INVOCATION_WHISPER
+	spell_requirements = NONE
+	cast_range = 6
+
+	active_msg = "You prepare to open your mind..."
+
+/datum/action/cooldown/spell/pointed/mind_gate/can_cast_spell(feedback = TRUE)
+	return ..() && isliving(owner)
+
+/datum/action/cooldown/spell/pointed/mind_gate/is_valid_target(atom/cast_on)
+	return ..() && ishuman(cast_on)
+
+/datum/action/cooldown/spell/pointed/mind_gate/cast(mob/living/carbon/human/cast_on, mob/living/carbon/human/owner)
+	. = ..()
+	if(cast_on.can_block_magic(antimagic_flags))
+		to_chat(cast_on, span_notice("Your mind feels closed."))
+		to_chat(owner, span_warning("Their mind doesn't swing open, but neither does yours."))
+		return FALSE
+
+	cast_on.adjust_confusion(10 SECONDS)
+	cast_on.adjustOxyLoss(30)
+	cast_on.cause_hallucination(\
+			get_random_valid_hallucination_subtype(/datum/hallucination/body), \
+			"Mind gate, cast by [owner]", \
+		)
```

Comment:
why do you somtimes use \, and sometimes not? also why is the indentation different here then on a list?

Reply:
If there is \ in some place that looks weird then its most likely me doing something earlier and then later changing it and just forgetting to clean it up. This for example used to be a bit more complex and the \ was used to break it up.